generate-db.js

```diff
@@ -3,6 +3,7 @@ import { glob, readFile, writeFile } from "node:fs/promises";
 import { gunzip } from "node:zlib";
 import { join } from "node:path";
 import { promisify } from "node:util";
+import assert from "node:assert/strict";
```

Comment:
How come this is being imported without braces?

Reply:
It seems to be a `default export`. If I add braces, TS complains